include/configs/imx8mq_evk.h

```diff
@@ -31,7 +31,6 @@
 /* For RAW image gives a error info not panic */
 #define CONFIG_SPL_ABORT_ON_RAW_IMAGE
 
-#undef CONFIG_DM_MMC
```

Comment:
Commit title is incorrect here:
`[FIO internal] imx8mq_evk: disable DM_I2C for SPL build`

Reply:
ah sorry, fixed and re-pushed